src/include/main/settings.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "common/types/value.h"
+#include "main/client_context.h"
+
+namespace kuzu {
+namespace main {
+
+struct ThreadsSetting {
+    static constexpr const char* name = "threads";
+    static constexpr const common::LogicalTypeID inputType = common::LogicalTypeID::INT64;
```

Comment:
I wonder if LogicalType makes more sense. Though these setting will almost always be non-recursive type, it's still good to use LogicalType.

Reply:
The inputType is a constexpr, we can't construct a logicalType here.
Also, duckdb uses logicalTypeID rather than logicalType here.
so i think it is ok to use typeid